io/eolearn/tests/test_io.py

```diff
@@ -257,8 +256,7 @@ def test_return_type(self):
     def test_time_interval(self):
         for task in self.task_cases:
             with self.subTest(msg='Test case {}'.format(task.name)):
-                self.assertEqual(task.eop.meta_info['time_interval'],
-                                 [iso_to_datetime(x) for x in ('2017-1-1', '2018-1-1')])
+                self.assertEqual(task.eop.meta_info['time_interval'], ('2017-1-1', '2018-1-1'))

```

Comment:
I don't understand, shouldn't `('2017-1-1', '2018-1-1')` in this test now be instances of `datetime`?

Reply:
It can be either.